kythe/cxx/verifier/verifier_main.cc

```diff
@@ -53,6 +53,10 @@ ABSL_FLAG(std::string, goal_regex, "",
 ABSL_FLAG(bool, convert_marked_source, false,
           "Convert MarkedSource-valued facts to subgraphs.");
 ABSL_FLAG(bool, show_anchors, false, "Show anchor locations instead of @s");
+ABSL_FLAG(bool, show_vnames, true,
```

Comment:
I wonder if these flag names (as well as other graphviz-related flags) ought to have a `dot_` or `graphviz_` (or `gv_`) prefix

Reply:
Possibly? In theory they could apply to the JSON dumps as well (but don't currently).